### PR TITLE
ID#11368  Ignite your Oracle Cloud Infrastructure Services - ADVANCED - new changes as requested

### PR DESCRIPTION
**Pull request by csmanea**
NEW WORKSHOP COMMENT -  ID#11368  Ignite your Oracle Cloud Infrastructure Services - ADVANCED

<< Requested changes>>
HI Workshop Team, Please make these changes to the workshop.
•	Could you move the Get Started Lab after introduction lab 
•	 Make the “Revisit Ignite Basics” lab 1 and then update the rest the labs number in manifest.json file
•	In Lab 3: Rename Want to Learn More to Learn More

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: oci-ignite-advanced/intro/intro.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+* [Oracle Cloud basics](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/concepts.htm)
+* [Connecting to a compute instance](https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/accessinginstance.htm)
+* Your **Oracle Cloud Account** - During this workshop, we will create a Level 100 environment for you to use on your tenancy.
+
+
+
+
+### You may now proceed to the next lab.
+
+Click on the next lab in this workshop to get started.
```
Comment (anooshapilli) [inline oci-ignite-advanced/intro/intro.md:41]: Delete this line.

File: oci-ignite-advanced/intro/intro.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* [Familiarity with OCI console](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/console.htm)
+* [Overview of Networking](https://docs.oracle.com/en-us/iaas/Content/Network/Concepts/overview.htm)
+* [Oracle Cloud basics](https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/concepts.htm)
+* [Connecting to a compute instance](https://docs.oracle.com/en-us/iaas/Content/Compute/Tasks/accessinginstance.htm)
+* Your **Oracle Cloud Account** - During this workshop, we will create a Level 100 environment for you to use on your tenancy.
+
+
+
+
+### You may now proceed to the next lab.
```
Comment (anooshapilli) [inline oci-ignite-advanced/intro/intro.md:39]: Remove the ### from this line

File: oci-ignite-advanced/create-oke/create-oke.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+    
+    ```
+
+![Deploy a Hello World app](images/cloudshell-commands-deploy-app-oke.png " ")
+
+
+_Congratulations! You have successfully completed the lab._
+
+## Want to Learn More?
```
Comment (anooshapilli) [inline oci-ignite-advanced/create-oke/create-oke.md:137]: Rename it to Learn more

**Review by anooshapilli — COMMENTED** [review]
@csmanea - Please make the changes requested today and resolve the conflicting files issue.

File: oci-ignite-advanced/intro/intro.md (hunk 1 above)
Comment (csmanea) [reply oci-ignite-advanced/intro/intro.md:41]: done

File: oci-ignite-advanced/intro/intro.md (hunk 2 above)
Comment (csmanea) [reply oci-ignite-advanced/intro/intro.md:39]: done

File: oci-ignite-advanced/create-oke/create-oke.md (hunk 3 above)
Comment (csmanea) [reply oci-ignite-advanced/create-oke/create-oke.md:137]: done

**Comment by anooshapilli** [comment]
@csmanea - The workshop looks good. Could you please resolve the conflicting files to approve the PR?

**Pull request closed without merging**